Libraries/LibJS/Runtime/JSONObject.cpp

```diff
@@ -155,22 +158,27 @@ ThrowCompletionOr<Optional<String>> JSONObject::serialize_json_property(VM& vm,
     if (value.is_object()) {
         auto& value_object = value.as_object();
 
-        // a. If value has a [[NumberData]] internal slot, then
+        // a. If value has an [[IsRawJSON]] internal slot, then
```

Comment:
When we amend a function for a proposal, we add it's spec link above the function definition as well. So above, we would have both:
```c++
// 25.5.2.1 SerializeJSONProperty ( state, key, holder ), https://tc39.es/ecma262/#sec-serializejsonproperty
// 1.4.1 SerializeJSONProperty ( state, key, holder ), https://tc39.es/proposal-json-parse-with-source/#sec-serializejsonproperty
```

This makes it easier to track what has been changed by a proposal when we come back later to either update the proposal or handle its merge into the main ECMA-262 spec.